scripts/codecov.sh

```diff
@@ -39,7 +39,14 @@ if [[ "${JOB_TYPE}" != "local" ]]; then
        fi
        curl -sS https://codecov.io/bash -o "${ARTIFACT_DIR}/codecov.sh"
        bash <(cat "${ARTIFACT_DIR}/codecov.sh") -Z -K -f "${COVER_PROFILE}" -r "${REPO_OWNER}/${REPO_NAME}" ${REF_FLAGS}
+       if [ $? -ne 0 ]; then
```

Comment:
since this script is run with "set -o errexit" i think the script is going to exit as soon as the script invocation fails, before you get to check the error code.

I think you need to do a "set +e" or you need to `|| true` the invocation.

example of the `|| true` pattern:
https://github.com/openshift/kubernetes/blob/307cfa6bd4b725ccf8a2dc061d6fbc399a891865/hack/verify-gofmt.sh#L56

the set +e pattern:
https://github.com/openshift/origin/blob/461b731871a4aeeff8b28c92921f27e53c6803e9/hack/verify-golint.sh#L12-L21

Reply:
With these above checks in place, we do not require `set -o errexit` I believe. So removed it.